### ISSUE TITLE
Using Command Psql to import a dump

### ISSUE THREAD

**Issue by dev-mobile** [yandex-qatools/postgresql-embedded#16]
Hi,
there is the need of a feature to import a dump file into the created database by using the [Psql Command]() .
I would suggest something like in the [PostgresProcess.java](https://github.com/yandex-qatools/postgresql-embedded/blob/master/src/main/java/ru/yandex/qatools/embed/postgresql/PostgresProcess.java)

```
java
    /**
     * import into database from file
     */
    public void importFromFile(File file) {
        if (file.exists()) {
            runCmd(getConfig(), Psql, "", new HashSet<>(singletonList("import into " + getConfig().storage().dbName() + " failed")),
                    1000,
                    new String[]{
                            "-U", getConfig().credentials().username(),
                            "-d", getConfig().storage().dbName(),
                            "-h", getConfig().net().host(),
                            "-f", file.getAbsolutePath()
                    }
            );
        }
    }

```

Thank you

**Comment by smecsia** [comment]
LGTM. Send the PR, but I would ask you to implement the test for this feature first.

**Comment by smecsia** [comment]
Fixed in #15
